core/controllers/admin.py

```diff
@@ -631,3 +632,19 @@ def get(self):
             'data': extracted_answers
         }
         self.render_json(response)
+
+
+class SendDummyMailToAdminHandler(base.BaseHandler):
+    """This function handles sending test emails."""
+
+    @acl_decorators.can_access_admin_page
+    def post(self):
+        username = self.username
+        if feconf.CAN_SEND_EMAILS:
+            response = {
+                'msg': 'Success! Mail sent to admin.',
```

Comment:
You don't need all this stuff. Just do self.render_json({}) at the end. (In general, follow the example of the surrounding code in the file.)

Reply:
Done.